Mathlib/Algebra/Order/LatticeGroup.lean

```diff
@@ -62,9 +64,21 @@ lattice, ordered, group
 
 -- Needed for squares
 -- Needed for squares
-universe u
+universe u v
 
-variable {α : Type u} [Lattice α] [CommGroup α]
+variable {α : Type u} {β : Type v}
+
+-- A linearly ordered additive commutative group is a lattice ordered commutative group
+
+@[to_additive] -- see Note [lower instance priority]
+instance LinearOrderedCommGroup.to_covariant_class (α : Type u)
+  [LinearOrderedCommGroup α] : CovariantClass α α (· * · ) (· ≤ ·) :=
+{ elim := fun a _ _ bc => OrderedCommGroup.mul_le_mul_left _ _ bc a }
```

Comment:
This instance already exists more generally as `OrderedCommGroup.to_covariantClass_left_le`

Reply:
I've removed the instance, and the rest of the PR still works.